### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by stejoo** [AnarchoTechNYC/ansible-role-radicale#1]
Thanks for creating this and sharing it.

I was looking at your code because I am going to deploy a Radicale installation soon. Being able to reuse your code would save me some time. However I cannot find a license for the code you have written. Unfortunately that prevents me from copying, using and/or modifying your code.
For more information have a look at: https://choosealicense.com/no-permission/
The same website can also help you with choosing a suitable license.

Could you please choose a license and include it in a LICENSE file?
Thank you.

**Comment by fabacab** [comment]
Thanks for your interest in this work.

> Unfortunately that prevents me from copying, using and/or modifying your code.

As we are presumably interested in being technically accurate here, your statement quoted above is incorrect: it means that you are *legally* prevented from doing these things.

This raises an obvious question. What exactly are you interested in doing involving this project's codebase that you would like to seek legal protection for before you act?

**Comment by stejoo** [comment]
Technically correct. Although covering legal protection is not my main goal, I do try do keep it in mind. As a fellow Linux and Ansible user I'm more interested in sharing interesting code and contributing back. I have a dislike for license violations as I work with open source and free software myself. Too often I see others take code, modify it for their own goals and leaving it at that.

I came across this repository while at work. I'm a Linux consultant for [AT Computing](https://www.atcomputing.nl/). We're a small Unix/Linux consulting and training company based in the Netherlands.

A client approached us to implement a CardDAV solution and we chose Radicale for the task. I'm currently tasked with setting up that deployment. I often write Ansible roles and playbooks to perform installations for clients. Instead of immediately writing an Ansible playbook/role for Radicale based on the installation instructions myself I did a search to see if somebody else had already written something like this. Your role was one that came up. While reviewing this work I decided that I like it and I can use it for my needs as well.

To me this looks like it's intended to be open source software. But without a license stating it is I am not comfortable using this directly for my client's needs.
Forking the repository and publishing any changes I make is no problem. I can do that if the license you pick would require it, GPL or CC for example. And I would probably still do so in case you pick a non-copyleft license (for example MIT, BSD, ISC) and I made any significant changes to it.

TL;DR: please select a license for this code to clear up your intentions with it and how you would like others to care for it.
